src/main/java/io/cryostat/net/web/http/api/v2/CredentialDeleteHandler.java

```diff
@@ -136,11 +134,8 @@ public IntermediateResponse<Void> handle(RequestParameters params) throws ApiExc
                     .send();
 
             return new IntermediateResponse<Void>().statusCode(200);
-        } catch (FileNotFoundException e) {
+        } catch (IllegalArgumentException e) {
```

Comment:
What's the rationale here for throwing the unchecked exception and catching it?

Reply:
Cleaned up in latest commit.